plugins/filter/lib/lib.pro

```diff
@@ -0,0 +1,40 @@
+TEMPLATE = lib
+TARGET = voicecall-filter-plugin
```

Comment:
Since this isn't going under plugins directory, I'd consider it more of a library rather than a plugin. Just voicecall-filter or something?

Likewise if the namespace should be without the plugin part.

Reply:
That sounds ok. I've removed the plugin mention from the lib (paths, namespace…).